scripts/nouns/zkvote_contract.ts

```diff
@@ -0,0 +1,56 @@
+import {
+    ZKVote, ZKVote__factory, NvoteVerifier__factory,
+    TallyVerifier__factory,
+  } from "../types";
+  import { Signer, Contract } from "ethers";
```

Comment:
Whitespace in this file seems off.  All statements from here are indented, trailing whitespace, etc.